src/mon/PGMap.cc

```diff
@@ -2621,7 +2621,9 @@ void PGMap::get_health_checks(
 
   // REQUEST_SLOW
   // REQUEST_STUCK
-  if (cct->_conf->mon_osd_warn_op_age > 0 &&
+  // SLOW_OPS unifies them in mimic.
+  if (osdmap.require_osd_release < CEPH_RELEASE_MIMIC &&
+      cct->_conf->mon_osd_warn_op_age > 0 &&
```

Comment:
What about using `get_val<>` here?

Reply:
@shinobu-x that's not in the scope of this change.